src/components/panel/panel.js

```diff
@@ -806,6 +850,75 @@ MdPanelService.prototype.newPanelAnimation = function() {
 
 
 /**
+ * Creates a panel group and adds it to a tracked list of panel groups.
+ * @param groupName {string} Name of the group to create.
+ * @param config (!Object=) Specific configuration object that may contain the
+ *     following properties:
+ *
+ *   - `maxOpen` - `{number=}`: The maximum number of panels that are allowed
+ *     open within a defined panel group.
+ */
+MdPanelService.prototype.newPanelGroup = function(groupName, config) {
+  if (!this._groups[groupName]) {
+    config = config || {};
+    var group = {
+      panels: [],
+      maxOpen: config.maxOpen > 0 ? config.maxOpen : false
+    };
+    this._groups[groupName] = group;
+  }
```

Comment:
Agreed, this will also make it easier to extend the group config later to include group-wide settings for `clickOutsideToClose` and other panel config options.